### PR TITLE
set GOARM from 6 to 7 when cross_build GOARM=GOARM7

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

 /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref #1445 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by kadisi** [comment]
cc @chendave PTAL

File: hack/lib/golang.sh [hunk 1]
```diff
@@ -292,11 +292,11 @@ kubeedge::golang::cross_build_place_binaries() {
     local name="${bin##*/}"
     if [ "${goarm}" == "8" ]; then
       set -x
-      GOARCH=arm64 GOOS="linux" CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -ldflags "$ldflags" $bin
+      GOARCH=arm64 GOOS="linux" GOARM=${goarm} CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -ldflags "$ldflags" $bin
```
Comment (chendave) [inline hack/lib/golang.sh]: AARCH64 doesn't need this, and no such option, pls remove this.

Comment (kadisi) [reply hack/lib/golang.sh]: thanks @chendave ,done

**Pull request merged**